### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheTechRobo**
Readability improvements for the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +13,7 @@ PirateBox is a collection of scripts / programs that allows you to use your wire
 as a local network to share files and chat anonymously. For more information please visit
 http://daviddarts.com/
 
------ Info -----
```
Comment (TheTechRobo) [inline README.md]: Easier to see now

**Comment by MaStr** [comment]
Thank you, but is it possible that you update the content onto the latest URLs and use cases? The main piratebox  page is located at PirateBox.cc 
The install scripts is also obsolete.

**Comment by TheTechRobo** [comment]
No problemo! I'll do that right now.

**Comment by TheTechRobo** [comment]
No problemo! I will do that now

**Comment by TheTechRobo** [comment]
P.S. What do i do with the install scripts? I'm going to comment it for now.

**Comment by TheTechRobo** [comment]
UPDATE: I will be closing this PR, GitHub says that I didn't do any commits to my fork so new PR #202

**Comment by TheTechRobo** [comment]
I will be closing this PR, GitHub says that my branch "thetechrobo-patch-1" doesnt exist. 

I will be merging in #202, #202 is better anyway :)

**Comment by MaStr** [comment]
Thank you! If it is possible please make a separate pull request for the development branch

**Pull request closed without merging**